README.md

```diff
@@ -24,3 +24,54 @@ end
 3. Commit your changes (`git commit -am 'Add some feature'`)
 4. Push to the branch (`git push origin my-new-feature`)
 5. Create new Pull Request
+
+## Testing
+
+### Installing the Rubies
+
+For setting up a dev environment with all the Rubies on it, it is convenient to
+use the [rake-compiler-dev-box](https://github.com/tjschuck/rake-compiler-dev-box).
```

Comment:
I personally use [chruby](https://github.com/postmodern/chruby) and [ruby-install](https://github.com/postmodern/ruby-install).  There are many options, though -- rvm is still a widely used option as is rbenv/ruby-build.

Overall, we're agnostic to what kinds of tools contributors use (and I imagine not everyone on the RSpec core team uses chruby/ruby-install like I do), but like @JonRowe said I don't feel comfortable having a recommendation in the README that none of us core devs have used.